src/components/Header/components/MainNavigation/MainNavigation.jsx

```diff
@@ -1,5 +1,25 @@
+import classes from './MainNavigation.module.scss';
+
 function MainNavigation() {
-  return <div>MainNavigation</div>;
+  return (
```

Comment:
you have repeated links with different text. Move it to separete component like Link and use here array.map(less code)

Reply:
Done.